File: HeartForCharity/HeartForCharity.Model/Requests/OrganisationProfileInsertRequest.cs
namespace HeartForCharity.Model.Requests
{
    public class OrganisationProfileInsertRequest
    {
        public string Name { get; set; } = null!;
        public string? Description { get; set; }
        public string? ContactEmail { get; set; }
        public string? ContactPhone { get; set; }
        public int? AddressId { get; set; }
        public string? LogoUrl { get; set; }
        public int? OrganisationTypeId { get; set; }
    }
}

File: HeartForCharity/UI/heartforcharity_desktop/lib/model/requests/organisation_profile_insert_request.dart
class OrganisationProfileInsertRequest {
  final String name;
  final String? description;
  final String? contactEmail;
  final String? contactPhone;
  final int? addressId;
  final String? logoUrl;
  final int? organisationTypeId;

  OrganisationProfileInsertRequest({
    required this.name,
    this.description,
    this.contactEmail,
    this.contactPhone,
    this.addressId,
    this.logoUrl,
    this.organisationTypeId,
  });

  Map<String, dynamic> toJson() => {
        'name': name,
        'description': description,
        'contactEmail': contactEmail,
        'contactPhone': contactPhone,
        'addressId': addressId,
        'logoUrl': logoUrl,
        'organisationTypeId': organisationTypeId,
      };
}
